core/initializer.py

```diff
@@ -0,0 +1,93 @@
+#  Drakkar-Software OctoBot
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import copy
+
+from backtesting import backtesting_enabled
+from config import CONFIG_DEBUG_OPTION_PERF, TimeFrames, CONFIG_NOTIFICATION_GLOBAL_INFO, NOTIFICATION_STARTING_MESSAGE, \
+    CONFIG_NOTIFICATION_INSTANCE
+from evaluator.Util import AdvancedManager
+from evaluator.evaluator_creator import EvaluatorCreator
+from services import ServiceCreator
+from tools import get_logger
+from tools.notifications import Notification
+from tools.performance_analyser import PerformanceAnalyser
+from tools.time_frame_manager import TimeFrameManager
+
+
+class Initializer:
```

Comment:
I think it would be easier to read with _ prefixes for protected methods

Reply:
Good idea to wrap initalizations in one file :)

Reply:
Yes good idea ;)